README.md

```diff
@@ -119,40 +118,56 @@ Assuming you already have everything needed to install the apps, this is what yo
    export CK8S_FLAVOR=[dev|prod] # defaults to dev
    ```
 
-2. Then set the path to where the ck8s configuration should be stored and the PGP fingerprint of the key(s) to use for encryption:
+1. Then set the path to where the ck8s configuration should be stored and the PGP fingerprint of the key(s) to use for encryption:
```

Comment:
Not really, i just noticed that we started doind this in some other repos and some upgrade docs here.
It's just a matter of personal preference i guess since numbering lists in markdown is irrelevant.